test.sh

```diff
@@ -101,6 +101,12 @@ if [[ "$RUN" =~ "integration" ]] ; then
     args+=("--filter" "${INT_FILTER}")
   fi
 
+  # We want to setup our 'HSM' before we get into the python part
```

Comment:
This block probably also needs to be replicated to `start.py`. I originally wanted to keep it outside of startservers.py, but it probably does make sense to move it in there to actually cover this...